base/scene.py

```diff
@@ -22,5 +23,5 @@ def render(self):
     def handle_input(self):
         pass
 
-    def transition_to(self, scene_name):
-        self.scene_manager.transition_to(scene_name)
+    def transition_to(self, scene_name, **kwargs):
```

Comment:
The callback function for the consume is one of those.
Anything that is not "kept" in the context but merely for a single call should not be stored in game_context.
But I meant the "not know about" in the sense that we will likely require different variables for future scenes we haven't thought about yet and extending the amount of arguments to pass without using a "container" like kwargs we will end up with a big mess